### ISSUE TITLE
Terraform import is not working

### ISSUE THREAD

**Issue by sajildeh** [l-with/terraform-provider-ldap#87]
I'm trying to use this module on a pre existing ldap, and when I do import (sing terrafomr import module) it fail, it's working with me for version 0.7.0 but after that all the version fails to do the import.
Can you please assist here?

**Comment by cah-chethan-hn** [comment]
> I'm trying to use this module on a pre existing ldap, and when I do import (sing terrafomr import module) it fail, it's working with me for version 0.7.0 but after that all the version fails to do the import. Can you please assist here?

Can you please let me know how did you make it work for terraform import on 0.7.0 because when ever i tried terraform import am also getting error "Error: unexpected end of JSON input". If you could help me on import command which it worked for you earlier then it would be great.
Command what i am trying is -terraform import 'module.create_ldapuser["testuser@example.com"].ldap_entry.user_example' 'uid=testuser'

**Comment by sajildeh** [comment]
I'm using the import module like this 
```
import {
  to = ldap_entry.ldap_user["USER_CN"]
  id = "ENTRY_DN"
}
```

```
resource "ldap_entry" "ldap_user" {
  dn       = DN
  data_json = jsonencode({
	....
  })
}
```

You can use the command by passing the cn and the entryDN

**Comment by cah-chethan-hn** [comment]
Actually, am using terraform version 1.1.17 so this import module feature is not supported  . I tried upgrading my terraform version to 1.5+ but even after import module was not able to generate configuration file out when i execute below command .Not sure what is wrong with my config.
terraform init
terraform plan -generate-config-out=generated_resources.tf

**Comment by sajildeh** [comment]
This is the command you need to use to import:
terraform import 'ldap_entry.ldap_user["USER_CN"]' ENTRY_DN

**Comment by sajildeh** [comment]
@pablo-ruth @barbaluc @barryib Any update on the import issue?
Also I faced some other issues with my current module 0.7.0, when I imported and run apply the field for Multi OTP in DS was removed while it was under ignore. Can you please assist further

Regards,
Saji

**Comment by l-with** [comment]
should be fixed with version 0.10.0

**Comment by l-with** [comment]
> @pablo-ruth @barbaluc @barryib Any update on the import issue? Also I faced some other issues with my current module 0.7.0, when I imported and run apply the field for Multi OTP in DS was removed while it was under ignore. Can you please assist further
> 
> Regards, Saji

This should placed in another issue with a concrete example.
